test/jdk/java/lang/StringBuffer/Exceptions.java

```diff
@@ -65,60 +64,69 @@ private static void tryCatch(String s, Throwable ex, Runnable thunk) {
             pass(s);
     }
 
-    public static void main(String [] args) {
+    public static void main(String[] args) {
         System.out.println("StringBuffer()");
         tryCatch("  no args", null, new Runnable() {
-                public void run() {
-                    new StringBuffer();
-                }});
+            public void run() {
+                new StringBuffer();
+            }
+        });
```

Comment:
Nit: The above formatting (line 70-97) is inconsistent with the formatting in line 110-124.   It'd be good to use the same formatting.

Reply:
Hi Mandy, thanks for reviewing this. 

> I suggest to separate the client changes (both src and test) in a separate PR for the client review.

Does "client changes" means changes involving src/java.desktop and test/java/awt?

> src/java.base/share/classes/java/util/concurrent/CopyOnWriteArrayList.java needs to be updated in JSR 166 upstream repo. Better to file a separate issue for this change to ensure that gets fixed in the upstream project.

Can you please paste a link for that? I'm not sure where I can find JSR 166 upstream repo..

> Nit: The above formatting (line 70-97) is inconsistent with the formatting in line 110-124. It'd be good to use the same formatting.

Restored.

Reply:
> Does "client changes" means changes involving src/java.desktop and test/java/awt?

src/java.desktop, test/java/awt, and test/javax/imageio

Reply:
> > src/java.base/share/classes/java/util/concurrent/CopyOnWriteArrayList.java needs to be updated in JSR 166 upstream repo. Better to file a separate issue for this change to ensure that gets fixed in the upstream project.
> 
> Can you please paste a link for that? I'm not sure where I can find JSR 166 upstream repo..

What I meant is to file a JBS issue for this change and revert the change in this patch.   That can be fixed when the next JSR 166 changes are imported to JDK. 

I wasn't sure if this is the right repo:  http://gee.cs.oswego.edu/cgi-bin/viewcvs.cgi/jsr166/src/main/

Reply:
Okay. I've filed https://bugs.openjdk.java.net/browse/JDK-8270057 and https://bugs.openjdk.java.net/browse/JDK-8270058 for JSR 166 and client code respectively. These codes have been restored.

(Sorry for force-pushing, my mistake..)